assets/js/amp-block-editor-toggle.js

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
Ideally this file would be converted into a module similar to all of the scripts for blocks in https://github.com/Automattic/amp-wp/tree/develop/blocks

This needs to be done for https://github.com/Automattic/amp-wp/blob/develop/assets/js/amp-block-validation.js and https://github.com/Automattic/amp-wp/blob/develop/assets/js/amp-editor-blocks.js as well. This would allow for being able to use Babel to handle i18n and it would allow for the scripts to all be concatenated into a single build script.

Reply:
Sure, Issue #1298 now tracks bundling this `amp-block-editor-toggle.js` module with `amp-block-validation.js`. We could also add converting `amp-editor-blocks.js` to a module.